client/src/components/survey-edit/survey-edit-container.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../scss/variables';
+@import '../../scss/colors';
+@import '../../scss/mixins';
+@import '../../scss/atoms-organisms/button';
+
+.drag-helper {
+  @include z-level(1);
+}
+
+.add-button-container {
+  margin-top: 10px;
+}
+
+.add-button {
+  @include button();
+  background: $palette-light;
+  display: block;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  width: 100%;
+
+  &:disabled {
+    opacity: 0.5;
+  }
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  @include z-level(2);
+  &.first {
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    justify-content: center;
+    align-items: center;
+    background-color: rgba(0, 0, 0, 0.3);
+  }
+}
+
+.modal {
```

Comment:
are these duplicated styles somewhere, consider extending @joonalillfors

Reply:
this fine

Reply:
no duplications